src/workerd/api/sql.h

```diff
@@ -22,9 +22,10 @@ class SqlStorage final: public jsg::Object, private SqliteDatabase::Regulator {
 
   class Cursor;
   class Statement;
+  class IngestResult;
 
   jsg::Ref<Cursor> exec(jsg::Lock& js, kj::String query, jsg::Arguments<BindingValue> bindings);
-  kj::String ingest(jsg::Lock& js, kj::String query);
+  jsg::Ref<IngestResult> ingest(jsg::Lock& js, kj::String query);
```

Comment:
I know that `ingest` is still experimental so this should be fine, but I just want to confirm that we're sure changing the return value here won't break anyone that we know of?

Reply:
The only thing it'll break is the D1 worker, and I've already merged (and am working on deploying) code that handles both the old and the new type.